docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2096,6 +2097,65 @@ Result:
 }
 ```
 
+### getRecentPrioritizationFees
+
+Returns a list of minimum prioritization fees from recent blocks. Currently, a
+node's prioritization-fee cache stores data from up to 150 blocks.
+
+#### Parameters:
+
+- `<array>` - (optional) An array of account address strings. If this parameter is provided, the response will reflect the minimum prioritization fee to land transaction locking any of the provided accounts as writable.
```

Comment:
nit: `locking any of` or `locking all of`? Since return is Max(all prioritization fees). I can reason both be right, not sure which is more natural.

Reply:
Yeah, I think "any" is okay.. Client might use the call to query a superset of what one transaction needs, basically combining a few queries into one.

Reply:
```suggestion
- `<array>` - (optional) An array of account address strings. If this parameter is provided, the response will reflect the minimum prioritization fee to land a transaction locking any of the provided accounts as writable.
```
Missing word

Reply:
Pondering this more, I think "all" is actually more correct!
Clients can still choose to call a superset, but the data won't distinguish between blocks where different addresses are responsible for the min fee.